javascript/photo.js

```diff
@@ -51,12 +51,12 @@ backgrounds.Photo = new Model({
      * Generates a permutation of size N.
      */
     permutation: function(N) {
-        var numbers = Array();
-        var ind, temp;
+        var i, ind, temp, numbers;
+        numbers = [];
```

Comment:
[Question] What's the advantage of this over Array() ?

Reply:
No advantage.  They're the same object.  Just seemed like this was more conventional.  Either way is fine - I don't have a strong preference.